### PR TITLE
Esn restangular

### PR DESCRIPTION
**Pull request by MichaelBailly**

### CODE REVIEW

File: src/esn.inbox.libs/app/services/generate-jwt-token/generate-jwt-token.spec.js [hunk 1]
```diff
@@ -16,12 +16,12 @@ describe('The generateJwtToken service', function() {
     generateJwtToken = _generateJwtToken_;
   }));
 
-  it('should resolve response data on success', function(done) {
-    var responseData = { key: 'value' };
+  it.only('should resolve response data on success', function(done) {
```
Comment (rezk2ll) [inline src/esn.inbox.libs/app/services/generate-jwt-token/generate-jwt-token.spec.js:19]:
```suggestion
  it('should resolve response data on success', function(done) {
```

File: src/esn.inbox.libs/app/services/generate-jwt-token/generate-jwt-token.spec.js [hunk 2]
```diff
@@ -16,12 +16,12 @@ describe('The generateJwtToken service', function() {
...
+    var responseData = 'the server response';
```
Comment (rezk2ll) [inline src/esn.inbox.libs/app/services/generate-jwt-token/generate-jwt-token.spec.js:20]:
```suggestion
    const responseData = 'the server response';
```

**Review by rouazana — COMMENTED** [review]
did you check other frontends also?

**Comment by rouazana** [comment]
even in frontend-inbox there are other changes to do, in particular in `inbox-users-identities-api-client.service.js` and `inbox-forwardings-api-client.service.js` where inboxRestangular is used instead of esnRestangular, causing 401 on the matching requests.

**Comment by rouazana** [comment]
also this PR misses a good description, a ticket number and a good commit description

**Review by rezk2ll — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Review by rouazana — APPROVED** [review]
just a quick note: the commit description explains the "what" does the commit (which is pretty obvious here reading the commit) not the "why" (which is well explained in the ticket). Next times I would prefer that the commit comment speak a little more about the why, here for example "#256 Call /api/jwt/generate with esnRestangular which handles correctly authentication"

**Pull request merged**